http/peer-id-auth.md

```diff
@@ -172,6 +172,17 @@ protocol operates as follows:
    Authentication-Info: libp2p-PeerID sig="<base64-signature-bytes>" bearer="<base64-encoded-opaque-blob>"
    ```
 
+   The server MAY include an `expires` field which contains the expiry time of
+   the bearer token in [ISO-8601](https://www.iso.org/iso-8601-date-and-time-format.html)
```

Comment:
I think we should use [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339) instead. It's a subset of ISO 8601 with a focus on [simplicity](https://www.rfc-editor.org/rfc/rfc3339#section-5.5) and use over internet.

In JS, I believe `toISOString()` is RFC 3339 compliant.

Reply:
I think that's what I meant.  I'm all in favour of simplicity.